storage/task.go

```diff
@@ -0,0 +1,195 @@
+package storage
+
+import (
+	"database/sql"
+	"errors"
+	"log"
+	"time"
+
+	"final/task"
+)
+
+const taskLimit = 10
+
+func (s *Storage) Add(t *task.Task) (int, error) {
+	ins, err := s.db.Exec(
+		"INSERT INTO scheduler (date, title, comment, repeat) VALUES (:date, :title, :comment, :repeat)",
+		sql.Named("date", t.Date),
+		sql.Named("title", t.Title),
+		sql.Named("comment", t.Comment),
+		sql.Named("repeat", t.Repeat),
+	)
+	if err != nil {
+		log.Println("can't add task:", err)
+		return 0, err
```

Comment:
Лучше оборачивать и возвращать ошибки, а вызывающая сторона сама решит, что с этим делать. А лог не стоит писать в таких слоях
```suggestion
		return 0, fmt.Errorf("can't add task: %w", err)
```